src/SAML2/BridgeContainer.php

```diff
@@ -55,6 +56,10 @@ public function debugMessage($message, $type): void
             $message = $message->ownerDocument->saveXML($message);
         }
 
+        if (!is_string($message)) {
+            throw new RuntimeException("Debug message error: could not convert message to string.");
+        }
```

Comment:
Cool to add this feature. Maybe good to mention this in the changelog, as it changes the behavior of the bundle.